articles/app-service/monitor-instances-health-check.md

```diff
@@ -136,6 +136,11 @@ If you restart the instance and the restart process fails, you will then be give
 
 Windows applications will also have the option to view processes via the Process Explorer.  This gives you further insight on the instance's processes including thread count, private memory, and total CPU time.
 
+## Diagnostic information collection
+For Windows applications, you have the option to collect diagnostic information in the Health Check tab.  When an instance becomes unhealthy, you can collect a memory dump using a storage account to view more information on why your instance may be unhealthy.  Enabling this option will change exiting auto-heal configurations.  
+
+Once diagnostic collection is enabled, you can create or choose an existing storage account for your files.  If using an existing storage account, it must be in the same region as your application.  Once the storage account is created or chosen, you can save the changes.  Keep in mind that saving will restart your application.  Once saved, you can go to your storage account resource and view the memory dump.
```

Comment:
If using an existing storage account, it must be in the same region as your application. -> You can only select storage accounts in the same region as your application.

Remove "Once the storage account is created or chosen, you can save the changes."

Once saved, you can go to your storage account resource and view the memory dump. -> Once saved, if your site instances are found to be unhealthy after continuous pings, you can go to your storage account resource and view the memory dump.